cmd/query/app/querysvc/adjuster/otelattribute.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package adjuster
+
+import (
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+
+	"github.com/jaegertracing/jaeger/pkg/otelsemconv"
+)
+
+var otelLibraryKeys = map[string]struct{}{
+	string(otelsemconv.TelemetrySDKLanguageKey):   {},
+	string(otelsemconv.TelemetrySDKNameKey):       {},
+	string(otelsemconv.TelemetrySDKVersionKey):    {},
+	string(otelsemconv.TelemetryDistroNameKey):    {},
+	string(otelsemconv.TelemetryDistroVersionKey): {},
+}
+
+// OTELAttribute creates an adjuster that moves the OpenTelemetry library
+// attributes from spans to the parent resource.
+func OTELAttribute() OTELAttributeAdjuster {
```

Comment:
```suggestion
func ResourceAttributes() OTELAttributeAdjuster {
```
"OTEL" is already implied given the data model